book/src/testnets.md

```diff
@@ -1,10 +1,154 @@
 # Testnets
 
-Lighthouse does not offer a public testnet _yet_. In the meantime, it's easy to
-start a local testnet:
-
-- [Run a simple testnet](testnets.html) in Only Three CLI Commands™.
-- Developers of other Eth2 clients should see the [interop guide](interop.html).
-- The [sigp/lighthouse-docker](https://github.com/sigp/lighthouse-docker) repo
-	contains a `docker-compose` setup that runs a multi-node network with
-	built-in metrics and monitoring dashboards, all from your local machine.
+The Lighthouse CLI has a `testnet` sub-command to allow creating or connecting
+to Eth2 beacon chain testnets.
+
+For detailed documentation, use the `--help` flag on the CLI:
+
+```
+$ ./beacon_node testnet --help
+```
+
+```
+$ ./validator_client testnet --help
+```
+
+## Examples
+
+All examples assume a working [development environment](./setup.md) and
+commands are based in the `target/release` directory (this is the build dir for
+`cargo`).
+
+### Start beacon node given a validator count and genesis_time
+
+
+To start a brand-new beacon node (with no history) use:
+
+```
+$ ./beacon_node testnet -f quick 8 <GENESIS_TIME>
+```
+
+Where `GENESIS_TIME` is in [unix time](https://duckduckgo.com/?q=unix+time&t=ffab&ia=answer).
+
+> Notes:
+>
+> - This method conforms the ["Quick-start
+genesis"](https://github.com/ethereum/eth2.0-pm/tree/6e41fcf383ebeb5125938850d8e9b4e9888389b4/interop/mocked_start#quick-start-genesis)
+method in the `ethereum/eth2.0-pm` repository.
+> - The `-f` flag ignores any existing database or configuration, backing them
+>   up before re-initializing.
+> - `8` is the validator count and `1567222226` is the genesis time.
+> - See `$ ./beacon_node testnet quick --help` for more configuration options.
+
+### Start Beacon Node given a genesis state file
+
+A genesis state can be read from file using the `testnet file` subcommand.
+There are three supported formats:
+
+- `ssz` (default)
+- `json`
+- `yaml`
+
+Start a new node using `/tmp/genesis.ssz` as the genesis state:
+
+```
+$ ./beacon_node testnet --spec minimal -f file ssz /tmp/genesis.ssz
```

Comment:
Should we explain in the Notes what `minimal` refers to?

Reply:
Done :)